<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/product-icon.png</normaloff>icons/product-icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="top_bar">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 170, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="CadProd_Text">
         <property name="font">
          <font>
           <pointsize>25</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
</string>
         </property>
         <property name="text">
          <string>Cadastro de Produtos</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="background">
      <property name="minimumSize">
       <size>
        <width>459</width>
        <height>700</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(35, 35, 35);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_fake">
         <property name="minimumSize">
          <size>
           <width>419</width>
           <height>653</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(60, 60, 60);
border-radius: 8px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QFrame" name="paineldeobjetos">
            <property name="minimumSize">
             <size>
              <width>380</width>
              <height>580</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>380</width>
              <height>642</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QPushButton" name="pushButton_MCad">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>100</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	background-color: rgb(85, 170, 255);
}
QPushButton:hover{
	background-color: rgb(152, 202, 255);
}
QPushButton:pressed{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
}</string>
               </property>
               <property name="text">
                <string>Cadastrar produto</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_MBus">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>100</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	background-color: rgb(85, 170, 255);
}
QPushButton:hover{
	background-color: rgb(152, 202, 255);
}
QPushButton:pressed{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
}</string>
               </property>
               <property name="text">
                <string>Consultar produto</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_MPro">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>100</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	background-color: rgb(85, 170, 255);
}
QPushButton:hover{
	background-color: rgb(152, 202, 255);
}
QPushButton:pressed{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
}</string>
               </property>
               <property name="text">
                <string>Exibir todos os produtos</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_MExc">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>100</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	background-color: rgb(255, 42, 42);
}
QPushButton:hover{
	background-color: rgb(255, 144, 144);
}
QPushButton:pressed{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
}</string>
               </property>
               <property name="text">
                <string>Excluir produto</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
